src/stylesheets/toolbar.scss

```diff
@@ -70,3 +77,13 @@
     display: block;
   }
 }
+
+.out-of-support-block {
+  background-color: var(--color-admonition-important-bg);
+  border-color: var(--color-admonition-important);
+  color: var(--color-admonition-important-text);
+  text-align: center;
+  width:100%;
+}
```

Comment:
❓ This class is already defined in https://github.com/bonitasoft/bonita-documentation-theme/blob/d315394fd4414de15b8b2718b47407d51c17ad5e/src/stylesheets/doc.scss#L1092
Are we sure we need the duplication?

Reply:
Good catch !
I'll remove .out-of-support-block from doc.scss and move also .next-release-block in toolbar.scss

Reply:
fixed in 5b8df92